### ISSUE TITLE
unlock Twitter account

### ISSUE THREAD

**Issue by tomkralidis** [geopython/pygeoapi#395]
While completing the [pygeoapi](https://twitter.com/pygeoapi) Twitter profile, I entered the date of birth of the project as 2018-02-14 (first commit).  This caused Twitter to lock the account due to age restrictions.

I have sent in a note to Twitter to unlock the account.

Mea culpa - serves me right for trying to be smart with the profile!

**Comment by tomkralidis** [comment]
https://twitter.com/tomkralidis/status/1249306845925257218

**Comment by tomkralidis** [comment]
Update: discussions are ongoing with Twitter to unlock the account.

**Comment by tomkralidis** [comment]
Case # 0150498263

**Comment by tomkralidis** [comment]
Update: I tried logging in again and the account wasn’t recognized (like it never existed). So I re-created the account and everything seems to work as expected.  I added an opening tweet and a pinned tweet (we can change at any time).

Reminder: tweeting guidelines are at https://github.com/geopython/pygeoapi/wiki/TwitterAccount and the account is bound to PSC.

We are now live! https://twitter.com/pygeoapi
